docs/articles/documentation/test-api/A-Z.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: docs
+title: A-Z
+permalink: /documentation/test-api/a-z.html
+---
+# A-Z
+
+This topic contains an alphabetical list of all the test API functions and objects with members.
+
+## Functions
+
+* [afterEach](test-code-structure.html#initialization-and-clean-up)
+* [beforeEach](test-code-structure.html#initialization-and-clean-up)
+* [fixture](test-code-structure.html#fixtures)
+* [page](test-code-structure.html#specifying-the-start-webpage)
+* [test](test-code-structure.html#tests)
```

Comment:
We only have two global functions - `fixture` and `test`.
The `page`, `afterEach` and `beforeEach` functions are exposed by `Fixture` objects in the global scope. (check this with @inikulin). I'm not sure but it seems we have to formally mention this `Fixture` objects in this topic.